### ISSUE TITLE
PRC Table 174 undefined bold term number_of_reference_points for point_reference_array

### ISSUE THREAD

**Issue by mvrhel** [pdf-association/pdf-issues#427]
**Describe the bug**
point_reference_array in Table 174 says that the decision as to whether or not the data is Huffman compressed is determined by number_of_reference_points=>3 (I would first change this to >=)   The issue is that number_of_reference_points is not defined in the document.  There is a term reference array_size which refers to the points_is_reference_array structure size.  I have seen files, that clearly have a points_is_reference_array of size 4 but point_reference_array compressed array is empty.  So, what is number_of_reference_points?

**Comment by mvrhel** [comment]
So I reviewed this as I had forgotten about it.  The issue is that number_of_reference_points  is not defined.    What I have found is the following: number_of_reference_points  is the number of non-zero elements in the points_is_reference_array.   This took some effort and luck in finding a couple files that were right on the edge with respect to the decision of the point_reference_array being huffman encoded or not.    To the best of my knowledge this is correct.  I of course am not the author of the original spec so I don't know this for sure.

**Comment by stuartgalt** [comment]
Proposal:  remove undefinded term and change the desctiption to:

![Image](https://github.com/user-attachments/assets/2f2d95a5-e6ce-4b99-8958-e2f6b9d4d797)

**Comment by mvrhel** [comment]
No.  We have to be careful here.  I would keep from previous version "Its value is implicit and set to number_of_reference_points>=3"  , where number_of_reference_points is the number of non-zero elements in the points_is_reference_array.    So add that extra where statement.

**Comment by mvrhel** [comment]
Yes.  That looks good.

**Comment by stuartgalt** [comment]
Closing - this seems to be fixed.
